File: Java/Jugador.java
import java.util.Scanner;

public class Jugador{
    private String nombre;

    public Jugador(String nombre){
        this.nombre = nombre;
    }

    public String getNombre(){
        return this.nombre;
    }

    public int escogerRana(){
        int rana;
        Scanner lector = new Scanner(System.in);
        do{
            System.out.println("Indique la rana para saltar [0-6]: ");
            rana = lector.nextInt();
        }while(rana < 0 || rana > 6);
        return rana;
    }

    public int escogerMovimiento(Rana rana, int posicion){
        int salto;
        Scanner lector = new Scanner(System.in);
        do{
            System.out.println("Indique: \n - 1 si la rana hace salto simple, o \n - 2 si la rana hace salto doble.");
            salto = lector.nextInt();
        }while(salto < 1 || salto > 2);
        if(rana.vaPara(">"))
            return posicion + salto;
        
        if(rana.vaPara("<"))
            return posicion - salto;

        return -1;
    }
}

File: Python/Jugador.py
from Rana import Rana

class Jugador:
    
    def __init__(self, nombre:str):
        self.nombre = nombre
        
    def get_nombre(self):
        return self.nombre
    
    def escoger_rana(self):
        print("Indique la rana para saltar [0-6]:")
        rana = int(input())
        if rana < 0 or rana > 6:
            rana = self.escoger_rana()
        return rana
    
    def escoger_movimiento(self, rana:Rana, posicion:int):
        print("Indique: \n - 1 si la rana hace salto simple, o \n - 2 si la rana hace salto doble.")
        salto = int(input())
        while salto < 1 or salto > 2:
            print("Indique: \n - 1 si la rana hace salto simple, o \n - 2 si la rana hace salto doble.")
            salto = int(input())
        
        if rana.va_para(">"):
            return posicion + salto
        if rana.va_para("<"):
            return posicion - salto
        
        return -1
            